packages/neutrino-middleware-hot/README.md

```diff
@@ -57,6 +57,7 @@ The following is a list of plugins and their identifiers which can be overridden
 | Name | Description | Environments and Commands |
 | --- | --- | --- |
 | `hot` | Enables Hot Module Replacement. | all |
+| `named-modules` | Show the relative path of the module to be displayed when HMR is enabled. | all |
```

Comment:
The description for the other usages of `NamedModulesPlugin` (from neutrino-middleware-chunk) use this string:

> Enables named modules for improved debugging and console output.

...I think it might be good to make them consistent?

Also, sadly when updating the package docs, we have to make identical edits to both the copy under `packages/<PACKAGE>/README.md` and `docs/{middleware,presets}/<PACKAGE>/README.md`. 

Finally in this case, because `neutrino-middleware-hot` is used by both `neutrino-preset-web` and `neutrino-preset-node`, those package docs will also need updating (with the slight catch that for NODE_ENV=production, the web preset already used `named-modules` via `neutrino-middleware-chunk`).